README.md

```diff
@@ -83,6 +83,20 @@ The language server accepts various settings through the `initializationOptions`
 | locale            | string   | The locale to use to show error messages. |
 | plugins           | object[] | An array of `{ name: string, location: string }` objects for registering a Typescript plugins. **Default**: []                                                                                                                                                         |
 | preferences       | object   | Preferences passed to the Typescript (`tsserver`) process. See below for more info.                                                                                                                              |
+| tsserver          | object   | Options related to the `tsserver` process. See below for more info.                                                                                                                              |
```

Comment:
what is the difference between `initializationOptions.logVerbosity`, `initializationOptions.tsserver.trace` and the "--tsserver-log-verbosity" cli flag?

Reply:
`initializationOptions.logVerbosity` and `--tsserver-log-verbosity` are related to file-based logs.

I have plans to do further cleanup here, removing some command line options and unifying some naming.

Reply:
Also I just realized that there is a difference between the term "log" and "trace". 
Log is just a text log, no way of knowing how you got to the log.
Trace should gives you information to trace something from the time till its requested till the time it is done.

Reply:
I'm trying to follow VSCode here as closely as I can to have feature and setting parity.

It's not entirely consistent there either.

For example there is a setting `typescript.tsserver.trace` which enables tsserver request logging to the panel and there is `typescript.tsserver.enableTracing` which enables creating performance tracing data to a file...

Reply:
In any case, do you see something that could be improved here or are you just stating your discoveries because I'm sure  :)

Reply:
I am just stating my discoveries :⁠-⁠D 

I would resolve the conversation, but sometimes Github doesn't show me the resolve conversation button.